### PR TITLE
show a message after cleanup with info

### PR DESCRIPTION
**Pull request by felineturtle67**
Shows a message after the cleanup with the amount of messages deleted

### Type

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature

### CODE REVIEW

**Comment by felineturtle67** [comment]
ok fixed an issue

File: redbot/cogs/cleanup/cleanup.py [hunk 1]
```diff
@@ -181,6 +182,7 @@ def check(m):
         log.info(reason)
 
         await mass_purge(to_delete, channel)
+        await ctx.send(f"Deleted {deleted} messages", delete_after=6)
```
Comment (NeuroAssassin) [inline redbot/cogs/cleanup/cleanup.py:185]: This needs to be i18n compatible.

File: redbot/cogs/cleanup/cleanup.py [hunk 2]
```diff
@@ -251,6 +253,7 @@ def check(m):
...
```
Comment (NeuroAssassin) [inline redbot/cogs/cleanup/cleanup.py:256]: This needs to be i18n compatible.

File: redbot/cogs/cleanup/cleanup.py [hunk 3]
```diff
@@ -292,6 +295,7 @@ async def after(
...
```
Comment (NeuroAssassin) [inline redbot/cogs/cleanup/cleanup.py:298]: This needs to be i18n compatible.

File: redbot/cogs/cleanup/cleanup.py [hunk 4]
```diff
@@ -339,6 +343,7 @@ async def before(
...
```
Comment (NeuroAssassin) [inline redbot/cogs/cleanup/cleanup.py:346]: This needs to be i18n compatible.

File: redbot/cogs/cleanup/cleanup.py [hunk 5]
```diff
@@ -391,6 +396,7 @@ async def between(
...
```
Comment (NeuroAssassin) [inline redbot/cogs/cleanup/cleanup.py:399]: This needs to be i18n compatible.

File: redbot/cogs/cleanup/cleanup.py [hunk 6]
```diff
@@ -429,6 +435,7 @@ async def messages(
...
```
Comment (NeuroAssassin) [inline redbot/cogs/cleanup/cleanup.py:438]: This needs to be i18n compatible.

File: redbot/cogs/cleanup/cleanup.py [hunk 7]
```diff
@@ -515,6 +522,7 @@ def check(m):
...
```
Comment (NeuroAssassin) [inline redbot/cogs/cleanup/cleanup.py:525]: This needs to be i18n compatible.

**Review by NeuroAssassin — CHANGES_REQUESTED** [review]
So, I think this is a PR that could prove to have some use in the future, so I wanted to offer a few points that need a few touch ups.

1. There needs to be a toggle to turn off the messages after the cleanup.  What the default is can be left to other people to discuss (but in my personal opinion would be best defaulted to off).
2. Any messages that are sent from the bot need to be i18n (internationalization) compatible, so that they can be translated and localized.  I've marked them below, and feel free to ask on the Red server if you need help with it.

Overall, I'm glad that you have found feature that can prove to be useful 👍.

**Pull request closed without merging**